pkg/component/observability/monitoring/persesoperator/crd_test.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
ℹ️ There are a few newly created files that bring back the year in the copyright license header that has been removed with: https://github.com/gardener/gardener/pull/12177

However, I wouldn't necessarily trigger CI/CD again for this PR unless you need to make changes anyway.
If it's still open after this one got merged, it could be combined with [my PR](https://github.com/gardener/gardener/pull/12214) or done in a follow-up.

```suggestion
// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```

Reply:
I doubt this is going through before Monday, so I simply pushed the rebased changes I had prepared. I'll restart the flaky tests as needed.